hathor/transaction/transaction.py

```diff
@@ -217,29 +218,34 @@ def get_sighash_all(self) -> bytes:
         if self._sighash_cache:
             return self._sighash_cache
 
+        sighash = self._get_sighash(inputs=self.inputs, outputs=self.outputs)
+        self._sighash_cache = sighash
```

Comment:
Maybe rename it to `_sighash_all_cache`?

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/911/commits/b6112976da77aa63fb70f2eb22b1616ea933eecd

Reply:
Thanks!